src/main/plugins/org.dita.pdf2/xsl/fo/tables.xsl

```diff
@@ -929,37 +929,20 @@ See the accompanying LICENSE file for applicable license.
           <xsl:for-each select="$current">
             <fo:table-cell xsl:use-attribute-sets="strow.stentry">
                 <xsl:call-template name="commonattributes"/>
-                <xsl:variable name="frame" as="xs:string">
-                    <xsl:choose>
-                        <xsl:when test="../@frame">
-                            <xsl:value-of select="../@frame"/>
-                        </xsl:when>
-                        <xsl:otherwise>
-                            <xsl:value-of select="$table.frame-default"/>
-                        </xsl:otherwise>
-                    </xsl:choose>
-                </xsl:variable>
+                <xsl:variable name="frame" as="xs:string" select="if (../@frame)
```

Comment:
This could be even more terse as `(../@frame, $table.frame-default)[1]` but I don't know if we want to start using a pattern like that. In e.g. JS you can do `a || b` for the same thing and IMO it's more readable.